pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Mistake Odyssey"
+Subtitle: "Git, Github, VS Code and Markdown for non-programmers"
+date: 2021-07-08T06:00:00+01:00
+type: Communication
+tags:
+  - Git
+  - GitHub
+  - VS Code
+  - Markdown
+cover: /img/posts/mistake-odyssey/mistake-odyssey-carolinie-cavalli.webp
+authors:
+  - name: Barbaño González
+    photo: /img/avatars/barbano.webp
+    link: www.linkedin.com/in/barbano-gonzalez-moreno
```

Comment:
```suggestion
    link: https://www.linkedin.com/in/barbano-gonzalez-moreno
```

It's important to include the protocol part (`https://`) in a URL. Otherwise, the link with your name will not work correctly. Try it yourself: https://deploy-preview-300--asyncapi-website.netlify.app/blog/mistake-odyssey. Click on your name and see what happens to the address bar of the browser.

Reply:
Yes, it does not work, thanks for the tip :)